packages/overlay/src/integrations/sentry/tabs/TracesTab.tsx

```diff
@@ -8,6 +8,7 @@ export default function TracesTab() {
     <SentryEventsContextProvider>
       <Routes>
         <Route path="/:traceId/:spanId" element={<TraceDetails />} />
+        <Route path="/:traceId/info" element={<TraceDetails />} />
```

Comment:
Why don't we split `TraceDetails` component into smaller pieces and create `TraceSpanView`, `TraceInfoView` and `TraceDetails` or `TraceView`? It would save us a bunch of conditionals and would make the code more re-usable

Reply:
Ideally TraceInfo and SpanDetail should be a component instead of a route. Because they are just a drawer above the TraceDetails page.

And yes, we should split the code.

I'll do this splitting rn.

Reply:
Hi @BYK I'll make this change in next PR.

To answer your query:
- Yes, its doesnt make sense to have 3 diff routes for Trace detail page. I have added the use of the route below.
 ` <Route path="/:traceId/:spanId" element={<TraceDetails />} />`  -  To open Span detail drawer
  `<Route path="/:traceId/info" element={<TraceDetails />} />`   -  To open Trace Info drawer
 `<Route path="/:traceId" element={<TraceDetails />} />`  -  Main Trace detail page
- As mentioned above ideally opening of drawer(span info view and trace info view) should be just a component handled by states and params(if needed).
- Otherwise the Trace detail already have children components added to it.